docs/contributing/CONTRIBUTING.md

```diff
@@ -0,0 +1,66 @@
+# Contributing

```

Comment:
Do we anticipate needing multiple files under the "contributing" directory? If not, maybe just put all these docs in the main `docs` folder for now.

Reply:
I was considering splitting the type of contributions into their own files but it's a bit overkill for this after all